### PR TITLE
Added a tool for managing libvirt VMs

### PR DESCRIPTION
**Pull request by inercia**
For example, we can do a `libvirt-tool snapshot` right after the bootstrap and then `rollback` every time we want to test an orchestration from scratch...

### CODE REVIEW

File: caasp-kvm/tools/libvirt-tool [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+Snapshots:
+
+    snap        Create snapshots
+    rollback    Rollback to the latest snapshots
+    flush       Remove all the snapshots
+
+Suspend/resume:
+
+    susupend    Suspend the VMs
```
Comment (ereslibre) [inline caasp-kvm/tools/libvirt-tool]: Typo, `suspend`

File: caasp-kvm/tools/libvirt-tool [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    resume      Resume the VMs
+
+USAGE
+)
+
+# Utility methods
+log()        { (>&2 echo ">>> [caasp-kvm] $@") ; }
```
Comment (kiall) [inline caasp-kvm/tools/libvirt-tool]: s/caasp-kvm/libvirt-tool/ I think?

**Comment by MalloZup** [comment]
@inercia can we merge this?
what prevent us to merge it? tia

File: caasp-kvm/tools/libvirt-tool (hunk 1 above)
Comment (MalloZup) [reply caasp-kvm/tools/libvirt-tool]: :+1:

**Comment by MalloZup** [comment]
@inercia i assume we don't need to wait for CI for it

**Pull request merged**